.github/lychee.toml

```diff
@@ -0,0 +1,20 @@
+verbose = "info"
+no_progress = true
+
+user_agent = "https://github.com/renovatebot/renovate/tree/main/.github/lychee.toml"
+
+exclude = [
+  "localhost",
+  "codepen.io",
+  "repology.org",
+  "https://github.com/foo/bar/",
+  "https://raw.githubusercontent.com/foo/bar/",
+  "https://developer.mend.io/api/mc/badges",
+  "https://hub.docker.com/r/renovate/renovate/",
+  "^https\\:\\/\\/github\\.com\\/renovatebot\\/renovate\\/issues\\?q\\=is\\:issue\\+is\\:open\\+sort\\:updated\\-desc\\+label\\:manager\\:pre\\-commit$",
+]
+
+accept = ["200", "429"]
```

Comment:
The [lychee docs, rate limiting](https://lychee.cli.rs/troubleshooting/rate-limits/) list steps to reduce the number of 429 responses. Accepting `429` rate limited responses is the last resort. So maybe we should try the other things the docs list?

| Method | Do we use it? | Notes |
| :---- | :--- | :--- | 
| Do not retry a failed link, with `--max-retries 0` | No | |
| Set lower `--max-concurrency` | No | Lower concurrency makes the action slower! |
| Use `GITHUB_TOKEN` | No | A personal token with no extra permissions will work to check public links. |
| Caching | Yes, 1 day | |
| Ignore certain sites or paths | Yes | | 
| Accept `429` responses | Yes | Last resort according to the lychee docs. |

Reply:
We have an additional step to remove `429` responses from the cache, so the next run they will be retried. Eventually, all the rate limited links will be       checked (during 24 hours).

Reply:
#### My summary of your comment

Here's how I understood your comment:

- We get lots of PRs per day. Either from Renovate bot, or us, or contributors.
- This means the link check runs at least once for each incoming PR.
- Because the link check runs often: we can swallow the `429` and "hope" that on the next run(s) the link checker gets a good response.
- Over the course of a day, we checked all rate-limited links.

#### One day cache long enough?

Is a day enough time to be reasonably sure we checked all the links? I mean, is it worth it to have a longer time to live for the cache? Say 2 days?

Reply:
1 day is ok